next-gen/.editorconfig

```diff
@@ -15,65 +15,78 @@ file_header_template = Copyright The OpenTelemetry Authors\nSPDX-License-Identif
+dotnet_naming_rule.constant_fields_should_be_pascal_case.symbols = constant_fields
+dotnet_naming_rule.constant_fields_should_be_pascal_case.style = pascal_case_style
 
-dotnet_naming_symbols.constant_fields.applicable_kinds            = field
-dotnet_naming_symbols.constant_fields.required_modifiers          = const
+dotnet_naming_symbols.constant_fields.applicable_kinds = field
+dotnet_naming_symbols.constant_fields.required_modifiers = const
 
-dotnet_naming_style.pascal_case_style.capitalization              = pascal_case
+dotnet_naming_style.pascal_case_style.capitalization = pascal_case
 
 # static fields should be s_PascalCase
-dotnet_naming_rule.static_fields_should_have_prefix.severity      = warning
-dotnet_naming_rule.static_fields_should_have_prefix.symbols       = static_fields
-dotnet_naming_rule.static_fields_should_have_prefix.style         = static_prefix_style
+dotnet_naming_rule.static_fields_should_have_prefix.severity = none
+dotnet_naming_rule.static_fields_should_have_prefix.symbols = static_fields
+dotnet_naming_rule.static_fields_should_have_prefix.style = static_prefix_style
 
-dotnet_naming_symbols.static_fields.applicable_kinds              = field
-dotnet_naming_symbols.static_fields.required_modifiers            = static
+dotnet_naming_symbols.static_fields.applicable_kinds = field
+dotnet_naming_symbols.static_fields.required_modifiers = static
 
-dotnet_naming_style.static_prefix_style.required_prefix           = s_
-dotnet_naming_style.static_prefix_style.capitalization            = pascal_case
+dotnet_naming_style.static_prefix_style.required_prefix = s_
+dotnet_naming_style.static_prefix_style.capitalization = pascal_case
 
 # private fields should be _PascalCase
-dotnet_naming_rule.pascal_case_for_private_fields.severity        = warning
-dotnet_naming_rule.pascal_case_for_private_fields.symbols         = private_fields
-dotnet_naming_rule.pascal_case_for_private_fields.style           = pascal_case_underscore_style
+dotnet_naming_rule.pascal_case_for_private_fields.severity = warning
+dotnet_naming_rule.pascal_case_for_private_fields.symbols = private_fields
+dotnet_naming_rule.pascal_case_for_private_fields.style = pascal_case_underscore_style
 
-dotnet_naming_symbols.private_fields.applicable_kinds             = field
-dotnet_naming_symbols.private_fields.applicable_accessibilities   = private
+dotnet_naming_symbols.private_fields.applicable_kinds = field
+dotnet_naming_symbols.private_fields.applicable_accessibilities = private
 
-dotnet_naming_style.pascal_case_underscore_style.required_prefix  = _
-dotnet_naming_style.pascal_case_underscore_style.capitalization   = pascal_case
+dotnet_naming_style.pascal_case_underscore_style.required_prefix = _
+dotnet_naming_style.pascal_case_underscore_style.capitalization = pascal_case
 
 # Remove unnecessary expression value
 dotnet_diagnostic.IDE0058.severity = none
 
+[*Tests.cs]
```

Comment:
suggestion: Drop a .editorconfig into /test root and put rules specific to tests there